velox/common/base/benchmarks/BitUtilBenchmark.cpp

```diff
@@ -141,6 +141,54 @@ BENCHMARK_RELATIVE(BM_scatterBits, n) {
   runScatterBits(n, false);
 }
 
+void BM_forEachBit(
+    uint32_t iterations,
+    size_t numBits,
+    std::optional<size_t> bitToClear = std::nullopt) {
+  folly::BenchmarkSuspender suspender;
```

Comment:
Can you pass kNumRuns * iterations as `iterations` here to avoid re-doing the setup and teardown?